packages/builders/package.json

```
@@ -7,9 +7,6 @@
   "repository": "https://github.com/Agoric/agoric-sdk",
   "scripts": {
     "build": "exit 0",
-    "build:add-STARS-proposal": "agoric run scripts/inter-protocol/add-STARS.js",
```

This is a nice simplification to the code. Note that it changes the CLI of this package and someone may be surprised when it no longer works. Consider a deprecation message.
```suggestion
    "build:add-STARS-proposal": "echo instead: agoric run scripts/inter-protocol/add-STARS.js",
```

---

We can also just leave this here in that case. We just won't have any callers within the tests

---

I'd rank options this way:
1. counsel the user in what to do instead
2. take it out
3. leave it (last because it doesn't get us closer to not needing these. the next script to be added someone will think they need to include it here)